data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -199,15 +154,14 @@ public Object process(org.apache.hadoop.hive.ql.lib.Node nd, Stack<org.apache.ha
 							outputTable = "TEMP_" + UUID.randomUUID();
 							outputTableList.add(outputTable);
 						}
-						if (outTableNodes.size() == 0) {
+						if (outTableNodes.isEmpty()) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)